projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/interact/tilling.json

```
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:tilling",
+  "text": [
+    {
+      "text": "别再在造锄头上浪费钻石了。"
```

```suggestion
      "text": "别老惦记你那【终极奉献】了。"
```
稍作引申，效果可能好点。不行就改成“别做钻石锄头了，浪费。”